lib/logic/models/user.js

```diff
@@ -11,7 +11,13 @@ User.prototype.getAvatarUrl = function() {
     return this.avatar;
 };
 
-User.prototype.getUserLinkJsx = function() {
+User.prototype.getUserLinkJsx = function(key) {
+    if (key) {
+        return (
+            <a href={this.url} target="_blank" key={key}>{this.name}</a>
+        );
+    }
+
     return (
         <a href={this.url} target="_blank">{this.name}</a>
```

Comment:
You don't need to have the sadsies duplication here. You can just do:
```
return (
  <a href={this.url} target="_blank" key={key}>{this.name}</a>
);
```

If the key is `undefined` because you didn't give it as an arg, it'll just ignore it.